apps/backend/src/models/audit.py

```
@@ -74,31 +83,34 @@ class DataCollectionAudit(Base):
     @property
     def success_rate(self) -> float:
         """Calculate success rate for this operation type"""
         return 1.0 if self.status == "success" else 0.0
-    
+
     @property
     def performance_score(self) -> float:
         """Calculate performance score based on duration and cache hit"""
-        if self.duration_ms is None:
+        # Access ORM attributes via Any to avoid ColumnElement typing in mypy
+        orm_self = cast(Any, self)
```

Ruff (0.12.2)

Add quotes to type expression in `typing.cast()`

Add quotes

(TC006)